client/src/game/ui/HanabiCard.ts

```diff
@@ -1054,14 +1058,14 @@ export default class HanabiCard extends Konva.Group implements NodeWithTooltip {
         return;
       }
       const hand = this.parent.parent as unknown as CardLayout;
```

Comment:
null check here, and check for this.parent before assigning this.parent.parent (?)

Reply:
no need, hand will be undefined if any of those happen